corehq/apps/geospatial/models.py

```diff
@@ -1,10 +1,12 @@
-from shapely.geometry import Point
+from dataclasses import dataclass
```

Comment:
Apologies, I may have snuck in a run of `isort`. I forgot to do that in a separate commit